File: cs50x/Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }

    while (n < 1 || n > 8);
    for (int h = 0; h < n; h++)
    {
        for (int space = 0; space + h < n - 1; space++)
        {
            printf(" ");
        }
        for (int line = 0; line < h + 1; line++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: cs50x/Week 6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("height: ")
    if height > 0 and height < 9:
        break

for i in range(1, height+1):
    print(f"{" "*(height-i)}{"#"*i}{" "*2}{"#"*i}")
